frontend/src/components/worksheets/NewRun/NewRun.jsx

```diff
@@ -241,6 +243,7 @@ class NewRun extends React.Component<{
         if (cpu) args.push(`--request-cpus=${cpu}`);
         if (gpu) args.push(`--request-gpus=${gpu}`);
         if (docker) args.push(`--request-docker-image=${docker}`);
+        if (queue) args.push(`--request-queue=${queue}`);

```

Comment:
As discussed, since backend removes tag= anyway, we don't strip the tag= beforehand on frontend